sql/driver/conn.go

```diff
@@ -186,6 +188,10 @@ func (c *conn) Send(call sqlwire.Call) (*rows, error) {
 			} else if datum.StringVal != nil {
 				t[j] = *datum.StringVal
 			}
+			if t[j] != nil && !driver.IsValue(t[j]) {
```

Comment:
`s/IsValue/IsScanValue/g`. I think you'll also have to fix the use of `StringVal` at that point.